dkb2homebankTest.py

```diff
@@ -10,23 +10,27 @@ def setUp(self):
         warnings.simplefilter("ignore", ResourceWarning)
 
     def testShouldConvertCashFile(self):
-        dkb2homebank.convert_DKB_cash('testfiles/cash.csv')
+        dkb2homebank.convert_DKB_cash('testfiles/cash.csv', 'cashHomebank.csv')
         lineNumber = sum(1 for line in open('cashHomebank.csv'))
         self.assertEqual(lineNumber, 2)
 
+    # def testShouldConvertCashFileAndWriteToAlternativeOutputDir(self):
```

Comment:
this is a really cool idea! I'll try to fix this today or tomorrow :)

Reply:
```
    def testShouldConvertCashFileAndWriteToAlternativeOutputDir(self):
        tmpdir = tempfile.gettempdir()
        dkb2homebank.convert_DKB_cash('testfiles/cash.csv', os.path.join("%s", "cashHomebank.csv" % tmpdir))
```

this yields an error for me I'm not sure how to fix right now, but it might be that I'm not that firm with this string substitution (I usually use f-strings, but that's not backwards-compatible, so I wanted to avoid them for this repo). Do you know how to fix it?

Reply:
I guess it has something to do with the type of `tmpdir` :thinking: seems to be Union[bytes, str]

Reply:
Yeah, the `os.path.join()` instruction looks off.

Try this instead:

```python
def testShouldConvertCashFileAndWriteToAlternativeOutputDir(self):
    tmpdir = tempfile.gettempdir()
    dkb2homebank.convert_DKB_cash('testfiles/cash.csv', os.path.join(tmpdir, "cashHomebank.csv"))
```

Reply:
Ah right. I was a bit overzealous there with the string substitution :joy:

Reply:
Should be resolved with the latest commit @hamvocke. Please check again. Tests run without errors or warnings on my side now :)